src/main/java/com/bandwidth/sqs/queue/RedrivePolicy.java

```diff
@@ -0,0 +1,23 @@
+package com.bandwidth.sqs.queue;
+
+
+import org.immutables.value.Value.Immutable;
+
+import java.text.MessageFormat;
+
+@Immutable
+public interface RedrivePolicy {
```

Comment:
re"drive"? What is is being redriven or is something put on a different "drive" (as in hard drive, mount point)?

Or is this "retry"?